tools/send-to-caas/send-utils.js

```diff
@@ -195,7 +195,7 @@ export const getOrigin = (fgColor) => {
     return originLC;
   }
 
-  if (window.location.hostname.endsWith('.hlx.page')) {
+  if (window.location.hostname.endsWith(`.${SLD}.page`)) {
```

Comment:
Do we really need to test for `SLD` here or would `endsWith('.page')` be enough? Same question for L277